tests/functional_small/solvers_linear/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ if(PRESSIO_ENABLE_TPL_EIGEN)
   add_serial_utest(${TESTING_LEVEL}_solvers_linear_eigen ${SRC1})
 endif()
 
-if(PRESSIO_ENABLE_TPL_KOKKOS)
+if(PRESSIO_ENABLE_TPL_TRILINOS)
```

Comment:
This change is necessary because the test calls `solveAllowMatOverwrite`.

This function is in an `#if PRESSIO_ENABLE_TPL_TRILINOS` block because it uses a teuchos lapack wrapper. So if we're only using Kokkos (and not Trilinos), this function is not found.